src/pages/help-centre/dmt5.tsx

```diff
@@ -18,87 +18,182 @@ const StyledListItem = styled.li<{ marginTop: string }>`
+                                key={1}
+                            />,
+                            <ExternalLink
+                                to={'/dbot/'}
+                                external
+                                target="_blank"
+                                rel="noopener noreferrer"
+                                key={2}
+                            />,
+                            <ExternalLink
+                                to={'https://bot.deriv.com/'}
+                                external
+                                target="_blank"
+                                rel="noopener noreferrer"
+                                key={3}
+                            />,
+                        ]}
+                    />
+                )}
+            </Text>
+            <StyledText>
+                <Localize
+                    translate_text="<0>Deriv X</0> and <1>Deriv MT5</1> offer CFD trading on a similar range of assets, where you're able to open positions with leverage and your potential profit is only known when you close your positions. It's riskier than digital options trading because while you may potentially gain a lot if you win, you may also lose a lot if you don't. Deriv X and Deriv MT5 are popular among our traders who enjoy the risks as part of the excitement of CFD trading."
+                    components={[
+                        <ExternalLink
+                            to={'/derivx/'}
+                            external
+                            target="_blank"
+                            rel="noopener noreferrer"
+                            key={0}
+                        />,
+                        <ExternalLink
+                            to={'/dmt5/'}
+                            external
+                            target="_blank"
+                            rel="noopener noreferrer"
+                            key={1}
+                        />,
+                    ]}
+                />
+            </StyledText>
+        </ArticleWrapper>
+    )
+}
+
+const DifferenceDMT5DTraderEU = ({ text }: ArticleProps) => {
+    return (
+        <ArticleWrapper>
+            <StyledHeader as="h4">{text}</StyledHeader>
+            <StyledText>
+                <Localize
+                    translate_text="<0>DTrader</0> offers multipliers trading on a range of underlying assets such as forex, cryptocurrencies, and derived. This platform allows you to open multipliers trades that offer the opportunity to multiply potential profit without risking more than your stake. You may find it more intuitive if you're new to the trading world."
+                    components={[
+                        <ExternalLink
+                            to={'/dtrader/'}
+                            external
+                            target="_blank"
+                            rel="noopener noreferrer"
+                            key={0}
```

Comment:
do we need `key` here?

Reply:
`key` is always used in `<Localize components={[]} />`. It seems that components property only accepts array of component and it expects `key` for each component. Do we need to refactor and remove `key` everywhere we have only one item in the array?

Reply:
Oh Ok I see, I didn't know that, then it's ok, thanks @mohammad-rework :pray: